util/sh/lib/error.sh

```diff
@@ -0,0 +1,9 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+# shellcheck shell=bash
+
+error() {
+    log "ERROR: $1" 1 >&2
+    exit 1
```

Comment:
do you want to propagate the error exit code? i.e., `exit $1`?

Reply:
Good point! It would be nice but `$1` is the error message to print, not the code. For now I think I'll just leave it as is, we can enhance it if needed.